xmake/core/sandbox/modules/io.lua

```diff
@@ -77,6 +77,16 @@ function sandbox_io.gsub(filepath, pattern, replace)
     return data, count
 end
 
+-- check if file exists
+function sandbox_io.exists(filepath)
+    local file, _ = io.open(filepath, "r")
+    if file then
+        file:close()
+        return true
+    end
+    return false
+end
+
 -- open file
```

Comment:
Xmake has a ready-made interface to use, we don't need to add redundant interfaces.

```lua
if os.isfile(filepath) then
end
if os.isdir(dir) then
end
is os.exists(file_or_dir) then
end
```